OpenRA.Platforms.Default/Shader.cs

```diff
@@ -107,7 +99,7 @@ public Shader(string name)
 				var log = new StringBuilder(len);
 				OpenGL.glGetProgramInfoLog(program, len, out _, log);
 				Log.Write("graphics", $"GL Info Log:\n{log}");
-				throw new InvalidProgramException($"Link error in shader program '{name}'");
+				throw new InvalidProgramException($"'{bindings.VertexShaderName}' and '{bindings.FragmentShaderName}'");
```

Comment:
Keeping "Link error in shader program..." or some other message to give context would be useful. Just some shader names in the message isn't sufficiently insightful.